### DISCUSSION TITLE
Is FIDO2 supported as an MFA assertion?

### DISCUSSION THREAD

**Discussion by jkleinbach** [bergie/passport-saml#949] [Q&A] [answered]
I've searched but not found an answer to what MFA assertions are supported.

**Comment by markstos** [comment, marked as answer]
The IdP implements MFA. If they want to send an assertion to indicate that the user used MFA, they can.

This library does not contain any MFA-specific code.

**Reply by jkleinbach** [reply]
Thank you!
